blogObject.js

```diff
@@ -0,0 +1,185 @@
+var blog = {};
+
+//blog.prototype.sortArticlesByDate = function () {
+
+function sortArticlesByDate (){
+  for (var i = 0; i < blog.articles.length; i++){
+    var date = blog.articles[i].publishedOn;
+    blog.articles[i].publishedOn = Date.parse(date);
+  }
+  blog.articles.sort(function(a,b){
+   return b.publishedOn - a.publishedOn;
+  });
+}
```

Comment:
The above lines would be better suited in the blog.js file. Or the below blog.articles separated into it's own file.